pcsx2/gui/Panels/GameDatabasePanel.cpp

```diff
@@ -430,15 +434,21 @@ bool Panels::GameDatabasePanel::WriteFieldsToDB() {
 	if (serialBox->GetValue().IsEmpty()) return false;
 
 	Game_Data game;
+	wxString com = compatBox->GetValue();
 	GameDB->findGame(game, serialBox->GetValue());
 
 	game.id = serialBox->GetValue();
 
-	game.writeString(L"Serial",		serialBox->GetValue());
-	game.writeString(L"Name",		nameBox->GetValue());
-	game.writeString(L"Region",		regionBox->GetValue());
-	game.writeString(L"Compat",		compatBox->GetValue());
-	game.writeString(L"[comments]",	commentBox->GetValue());
+	game.writeString(L"Serial", serialBox->GetValue());
+	game.writeString(L"Name", nameBox->GetValue());
+	game.writeString(L"Region", regionBox->GetValue());
+	if (com == "Unknown")          game.writeString(L"Compat", "0");
+	else if (com == "Nothing")     game.writeString(L"Compat", "1");
+	else if (com == "Intro")       game.writeString(L"Compat", "2");
+	else if (com == "Menu")        game.writeString(L"Compat", "3");
+	else if (com == "In-game")     game.writeString(L"Compat", "4");
+	else if (com == "Playable")    game.writeString(L"Compat", "5");
+	else if (com == "Perfect")      game.writeString(L"Compat", "6");
```

Comment:
Forward compatibility. 'Perfect' is already agreed to include. Its definition is just not fixed so far. There is some discussion in the forum including devs in a similar named thread.